applications/nrf_desktop/src/hw_interface/selector_hw.c

```diff
@@ -145,8 +145,16 @@ static int disable_interrupts_nolock(struct selector *selector)
 static void selector_isr(const struct device *dev, struct gpio_callback *cb,
 			 uint32_t pins_mask)
 {
-	uint8_t port = dev - gpio_dev[0];
 	struct selector *sel;
+	size_t port;
+
+	for (port = 0; port < ARRAY_SIZE(gpio_dev); port++) {
+		if (dev == gpio_dev[port]) {
+			break;
+		}
+	}
+
```

Comment:
Consider removing this blank line as this one logical code unit that searches for the selector index within the array of GPIO devices (for loop) and then checks if such an index was found (assert).